docs/sdks/client-sdks/android.md

```diff
@@ -34,6 +34,8 @@ CompletableFuture<EppoClient> eppoClientFuture = new EppoClient.Builder("YOUR_SD
     .buildAndInitAsync();
 ``` 
 
+Flags and other features become available to the SDK after initialization. 
```

Comment:
I'd just rephrase it to "The SDK has access to flag configurations once initialization is complete."